gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/constants/Constants.bal

```diff
@@ -5,7 +5,7 @@ const string GW_HOME = "GW_HOME";
 const string API_CREATE = "API_CREATE";
 const string API_UPDATE = "API_UPDATE";
 const string API_DELETE = "API_DELETE";
-const string API_PUBLISH = "API_PUBLISH";
+const string API_STATE_CHANGE = "API_STATE_CHANGE";fmain
```

Comment:
Fixed in https://github.com/wso2/carbon-apimgt/commit/f089d81b6be2a0e8ddf8e530317fe680abafd246